cmd/encryption-v1.go

```diff
@@ -739,10 +740,10 @@ func (li *ListPartsInfo) IsEncrypted() bool {
 // DecryptedSize returns the size of the object after decryption in bytes.
 // It returns an error if the object is not encrypted or marked as encrypted
 // but has an invalid size.
-// DecryptedSize panics if the referred object is not encrypted.
+// DecryptedSize throws error msg and exits (fatal) if the referred object is not encrypted.
```

Comment:
`throws error msg and exits (fatal)` should be `logs error msg and exits (fatal)`